docfx.json

```diff
@@ -436,6 +436,8 @@
         "_csharplang/proposals/csharp-7.3/*.md": "C# 7.3 specification proposals",
         "_csharplang/proposals/csharp-8.0/*.md": "C# 8.0 specification proposals",
         "docs/framework/wcf/**/**.md": "WCF",
+        "docs/framework/winforms/**/**.md": "Windows Forms",
+        "docs/framework/wpf/**/**.md": "WPF",
```

Comment:
There is also a related `docs/desktop-wpf` folder. Should it be in this list as well or it's maintained in a different way?

Reply:
We're moving towards adding suffixes to all areas here but doing with different folders each time to be able to spot some oddities. @Thraka can you look at the desktop-wpf suffixes in a separate PR?